.gitmodules

```diff
@@ -1,6 +1,10 @@
 [submodule "3rdparty/nvbench"]
 	path = 3rdparty/nvbench
 	url = https://github.com/NVIDIA/nvbench.git
+[submodule "3rdparty/hipbench"]
+        path = 3rdparty/hipbench
+        url = https://github.com/yiakwy-xpu-ml-framework-team/hipbench.git
```

Comment:
May we directly use https://github.com/ROCm/hipBench?

Reply:
It seems that you made some modification https://github.com/ROCm/hipBench/compare/main...yiakwy-xpu-ml-framework-team:hipbench:fix_rapids_cmake

Reply:
Yes, the AMD-AI repo is unaccessible outside. Yes once the fixed is done (I can merge it by myself, but it may create interests holder conflicts), I will update the submodule description.